ArduCopter/mode_auto.cpp

```diff
@@ -872,9 +865,10 @@ void Copter::ModeAuto::nav_guided_run()
 //      called by auto_run at 100hz or more
 void Copter::ModeAuto::loiter_run()
 {
-    // if not auto armed or motor interlock not enabled set throttle to zero and exit immediately
-    if (!motors->armed() || !ap.auto_armed || ap.land_complete || !motors->get_interlock()) {
-        zero_throttle_and_relax_ac();
+    // if not armed set throttle to zero and exit immediately
+    if (!motors->armed() || !ap.auto_armed || ap.land_complete) {
```

Comment:
maybe we could factorize this block, it is use 3 time

Reply:
Yes we probably could although it's an existing issue ([see master here](https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/mode_auto.cpp#L876)) so I don't think I will increase the scope of this PR any more.